misc/cf-support

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+
+export WORKDIR=/var/cfengine
+export BINDIR="$WORKDIR/bin"
+export non_interactive=0
+if [ "$1" = "-M" ]; then
```

Comment:
You should also check `$#` here.

Reply:
For what reason would I check `$#`? There are two options supported and not in combination. If I check for number of arguments in relation to `-M` then isn't that the same difference as the current check of `[ "$1" = "-M" ]`?

Reply:
If there are no arguments, this will fail because `=` is a binary operator.

Reply:
Not sure I understand. The way I understand things and how it seems to test for me is that if no arguments then the expression will still be valid as `[ "" = "-M" ]` (two arguments given to a binary operator `=`) and give the right answer. I put quotes around `$1` to allow for no arguments.

```
[craig@mbp ~]$ x=""
[craig@mbp ~]$ [ $x = "M" ]
-bash: [: =: unary operator expected
[craig@mbp ~]$ [ "$x" = "M" ]
[craig@mbp ~]$ echo $?
1
```

Reply:
:+1: Cool. I though it would/might fail, maybe based on some past experience with a weird/old shell.

Reply:
Right on. It will definitely fail if no quotes because then you would end up with `[ = "-M" ]` like you said, binary versus unary operator. :tada: thanks for watching out for it!

Reply:
yes, in some places in our buildscripts we have things like `if [ "x$var" = "xtrue" ]` - but I believe it's for some very exotic exotics, maybe we even don't support them anyway (but I'm not sure, I haven't tested it).